monitor-a-dm-cluster: add more metrics (#768)
pingcap/docs-dm: en/monitor-a-dm-cluster.md

| Metric name | Description | Alert | Severity level |
|:----|:------------|:----|:----|
| remaining time to sync | The predicted remaining time it takes `syncer` to be completely migrated with the master (in minutes) | N/A | N/A |

> | replicate lag gauge | The latency time it takes to replicate the binlog from master to `syncer` (in seconds) | N/A | N/A |

Reviewer: ```suggestion
| replicate lag gauge | The latency time it takes to replicate the binlog from upstream to downstream` (in seconds) | N/A | N/A |
```

Reviewer: seems other old description is not right ...

Author: fixed in https://github.com/pingcap/docs-dm/pull/768/commits/8153636bdfd9e75c5b20a46ed439a1a1564eb995。 PTAL~

> | replicate lag histogram | The histogram of replicating the binlog from master to `syncer` (in seconds). Note that due to different statistical mechanisms, errors exist in the data | N/A | N/A |

Reviewer 2: maybe use Inaccuracy instead of errors here is better ?

Author: Yes you're right~ Error usually means an error message in computing. Fixed in https://github.com/pingcap/docs-dm/pull/768/commits/443a1023aec8af1b12bfe17bf5de253389e24535. Thx!

> | replicate lag histogram | The histogram of replicating the binlog from master to `syncer` (in seconds). Note that due to different statistical mechanisms, errors exist in the data | N/A | N/A |

Author: ```suggestion
| replicate lag histogram | The histogram of replicating the binlog from master to `syncer` (in seconds). Note that due to different statistical mechanisms, the data might be inaccurate | N/A | N/A |
```
